src/guides/v2.3/test/unit/writing_testable_code.md

```diff
@@ -27,18 +27,18 @@ We strongly recommend you do *not*:
 *  Use `new` to instantiate new objects, because that removes the flexibility the Magento dependency configuration offers.
 *  Use the `ObjectManager` directly in production code.
 
-There always is a better alternative, usually a [generated]({{ page.baseurl }}/extension-dev-guide/code-generation.html) `Factory` class, or a [`Locator`](https://thephp.cc/news/2015/09/dependencies-in-disguise){:target="_blank"} class of sorts.
+There always is a better alternative, usually a [generated] `Factory` class, or a [`Locator`][Locator]{:target="_blank"} class of sorts.
 
  {:.bs-callout-info}
-This rule applies only to production code. When writing [integration tests]({{ page.baseurl }}/test/integration/integration_test_execution.html), this is not true. In fact, the object manager is recommended for integration tests.
+This rule applies only to production code. When writing [integration tests][integration-tests], this is not true. In fact, the object manager is recommended for integration tests.
 
 ### Collaborator classes
 
 Whenever an external class property, class constant, or a class method is used in a file, this file depends on the class containing the method or constant. Even if the external class is not used as a instantiated object, the current class is still hard-wired to depend on it.
 
-[PHP](https://glossary.magento.com/php) cannot execute the code unless it can load the external class, too. That is why such external classes are referred to as *dependencies*. Try to keep the number dependencies of to a minimum.
+[PHP] cannot execute the code unless it can load the external class, too. That is why such external classes are referred to as *dependencies*. Try to keep the number dependencies of to a minimum.
```

Comment:
```suggestion
[PHP][PHP] cannot execute the code unless it can load the external class, too. That is why such external classes are referred to as *dependencies*. Try to keep the number dependencies of to a minimum.
```

Reply:
![image](https://user-images.githubusercontent.com/39480008/82633172-17c29e00-9c18-11ea-9073-4eac1bf9cc79.png)
If link defination name and text is same then we don't need to specify the link defination name

Reply:
That's right, we are good to go then. 

Thank you